Makefile

```diff
@@ -1,19 +1,19 @@
+img := gcr.io/istio-testing/build-tools:2019-07-31
```

Comment:
woudl be nice to break out the HUB seperately as a followup to permit third parties to build their own ci in air-gapped mode.